beta/tests/test_sota_checkpoints.py

```diff
@@ -316,18 +332,31 @@ def read_metric(metric_file_name: str):
                     batch = model_dict[model_name].get('batch', {})
                 else:
                     batch = None
-                if model_dict[model_name].get('mean_value', {}):
-                    mean_val = model_dict[model_name].get('mean_value', {})
-                else:
-                    mean_val = '[123.675,116.28,103.53]'
-                if model_dict[model_name].get('scale_value', {}):
-                    scale_val = model_dict[model_name].get('scale_value', {})
-                else:
-                    scale_val = '[58.4795,57.1429,57.4713]'
+                mean_val = model_dict[model_name].get('mean_value', {}) if not None else None
+                scale_val = model_dict[model_name].get('scale_value', {}) if not None else None
+                reverse_input_channels = model_dict[model_name].get('reverse_input_channels') if not None else None
```

Comment:
@pfinashx please add valid 'mean_value', 'scale_value', etc from archive with mo configs I shared with you.

Reply:
json updated